crates/bevy_animation/src/lib.rs

```diff
@@ -1220,6 +1228,25 @@ pub fn animate_targets(
         });
 }
 
+/// Specifies query for animation systems.
+/// It allows to specify which Time resource to use based on the [`bevy_ecs::query::QueryFilter`].
+/// That can be especially useful when you want to use a custom time resource for subset of animations.
+/// By default, there is added a one version with Time<()> and Without [`DontUseDefaultAnimationTime`] component.
+pub fn specify_animation_system<T: Default, F: bevy_ecs::query::QueryFilter + 'static>(
```

Comment:
I don't think "specify" is the right word here. I think this would be a lot clearer if this was an ordinary struct-based plugin called `TimeDependentAnimationPlugin` or something